### PR TITLE
OEL-1893: Move to oe_content version 3.

### PR DESCRIPTION
**Pull request by drishu**

### CODE REVIEW

File: composer.json
Comment (donquixote) [inline composer.json:27]: Given that this is require-dev, the main effect will be to determine which version will be tested.
There is no immediate effect on 3rd party projects, but it means future development on oe_starter_content can break oe_content 2.x compatibility.

I don't like the idea in oe_content that 2.x is no longer supported, while we have no stable release in 3.x. If this is the final word for oe_content, then the choice here seems ok.

An alternative might have been to try to support both versions temporarily, and do some conditional stuff to remove the patch from oe_content (*), and conditionally apply our own patches instead. I think I saw this somewhere, but I don't remember how it can be done.


_(*) Background: The composer.json of oe_content 2.x asks to apply a patch to inline_entity_form, which no longer works._

**Review by donquixote — APPROVED** [review]
I am approving this, but still feel a bit skeptical about it. Curious what @brummbar is going to say.

**Review by brummbar — APPROVED** [review]

File: composer.json
Comment (brummbar) [reply composer.json:27]: oe_content 2.x is no longer actively maintained by core team, all the new work goes in 3.x .

**Pull request merged**
